packages/env-loader/src/index.ts

```diff
@@ -128,8 +128,6 @@ async function cleanup(verbose = false) {
 
   const log = verbose ? console.log : () => {}
 
-  console.log('verbose:', verbose)
-
```

Comment:
Again, not actually related, but I noticed I forgot to clean up this `cleanup()`. 🫠